server/src/main/java/org/apache/druid/metadata/PendingSegmentRecord.java

```diff
@@ -32,6 +30,15 @@
 import javax.annotation.Nullable;
 import java.sql.ResultSet;
 
+/**
+ * Representation of a record in the pending segments table. <br/>
+ * Mapping of column in table to field:
+ * <li/>  id -> id (Unique identifier for pending segment)
+ * <li/>  sequence_name -> sequenceName (sequence name used for segment allocation)
+ * <li/>  sequence_prev_id -> sequencePrevId (previous segment id used for segment allocation)
+ * <li/>  upgraded_from_segment_id -> upgradedFromSegmentId (Id of the root segment from which this was upgraded)
+ * <li/>  task_allocator_id -> taskAllocatorId (Associates a task / task group / replica group with the pending segment)
```

Comment:
Does this render correctly as a javadoc?
The `li` tags should ideally open and close like standard HTML tags: `<li>content</li>`

The mapping of column name to field is not needed as it is pretty self explanatory.

Also, the field descriptions should just be added in the respective getters and/or  the constructor rather than at the class level.

Reply:
Also, all of the `<li></li>` need to be inside an enclosing `<ul></ul>` or `<ol></ol>`, same as HTML.

Reply:
Done